runtime/services/lightd/effects.js

```diff
@@ -293,8 +306,15 @@ LightRenderingContext.prototype.pixel = function (pos, r, g, b, a) {
   if (this._getCurrentId() !== this._id) {
     return
   }
-  a = applyAlphaFactor(a)
-  return light.pixel(pos, r, g, b, a)
+  var realA = applyAlphaFactor(a)
+  if (pos > 0 && pos < this.ledsConfig.leds) {
+    realTimeRGBData[pos].r = r
+    realTimeRGBData[pos].g = g
+    realTimeRGBData[pos].b = b
+    realTimeRGBData[pos].inputA = a
+    realTimeRGBData[pos].realA = realA
```

Comment:
Now it's the time to add the class `Color`, and move this transition inside.

Reply:
> Now it's the time to add the class `Color`, and move this transition inside.

Yeah, this maybe takes some times.

Reply:
Is this urgent? If no, it might be required.